doc/api/timers.md

```diff
@@ -94,6 +106,9 @@ set to `1`.
 If `callback` is not a function, a [`TypeError`][] will be thrown.
 
 ### setTimeout(callback, delay[, ...arg])
+<!-- YAML
+added: v0.0.1
```

Comment:
Doesn't seem quite right?

I'm pretty sure Isaac added these but that wasn't at the start.

The oldest reference I seem to be able to find of it exposed on the global (actually `GLOBAL` at the time) is https://github.com/nodejs/node/commit/7a2e784ad7386771d15249d446c914fba1675068 (v0.1.16)

Indeed it looks like the original was added before v0.0.1 but I'm not sure it was exposed in a way that uh, counts?

.. also note that (awkwardly) these were actually added to the timers module pretty long after they were available on the global object. They were added to the module in https://github.com/nodejs/node/commit/79944006e252170933e35862bdff2f7fba6bd762 (v0.3.1)